Sources/XMTPiOS/Client.swift

```diff
@@ -406,12 +410,11 @@ public final class Client {
 	{
 		let inboxId: String? =
 			allowReassignInboxId
-			? nil : try await inboxIdFromAddress(address: newAccount.address)
+			? nil : try await inboxIdFromIdentity(identity: newAccount.identity)
 
 		if allowReassignInboxId || (inboxId?.isEmpty ?? true) {
-			let signatureRequest = try await ffiAddWallet(
-				addressToAdd: newAccount.address.lowercased())
-
+			let signatureRequest = try await ffiAddIdentity(
+				identityToAdd: newAccount.identity)
```

Comment:
`testAddAccountsWithExistingInboxIds` in ClientTests was failing, looks like this fixes it:

```suggestion
			let signatureRequest = try await ffiAddIdentity(
				identityToAdd: newAccount.identity,
                                 allowReassignInboxId: allowReassignInboxId
                          )
```

Reply:
fixed in 7c38d2cafbdf27c271d229a2570aaaef4fcd9c0f